src/webui/www/public/css/style.css

```diff
@@ -484,3 +484,9 @@ td.statusBarSeparator {
 .statisticsValue {
     text-align: right;
 }
+
+/* iOS iframe scrolling */
+.downloadFrame .mochaContentWrapper, .uploadFrame .mochaContentWrapper {
+    overflow: auto !important;
```

Comment:
dumb question, does it work without `!important`?

also the comment above seems out-of-place, because this section applies to all browsers.

Reply:
I've actually identified an issue with this change. Will update this PR within the next few days.

Reply:
There is conflict now.